### PR TITLE
Update 3d_performance_and_limitations.rst

### PR DESCRIPTION
**Pull request by jmruzik87**
Couple of minor spelling fixes.
I thought the last sentence read oddly, so I changed it a bit.  Take it as you will.  I'm really getting interested in the engine and just trying to learn  and help :)

### CODE REVIEW

File: learning/features/3d/3d_performance_and_limitations.rst [hunk 1]
```diff
@@ -27,10 +27,10 @@ usability and flexibility. Some practical examples of this are:
 
 And there are many more examples of this! Game engines strive to be
 general purpose in nature, so balanced algorithms are always favored
-over algorithms that might be the fast in some situations and slow in
+over algorithms that might be the fastest in some situations and slow in
 others.. or algorithms that are fast but make usability more difficult.
 
-Godot is not an exception and, while it is designed to have backends
+Godot is not an exception, and while it is designed to have backends
```
Comment (akien-mga) [inline learning/features/3d/3d_performance_and_limitations.rst]: That comma was correct AFAIK.

Comment (cbscribe) [reply learning/features/3d/3d_performance_and_limitations.rst]: I would say there shouldn't be a comma at all. 

But there are a lot of grammar issues in this doc. It really needs a full review.

File: learning/features/3d/3d_performance_and_limitations.rst [hunk 2]
```diff
@@ -27,10 +27,10 @@ usability and flexibility. Some practical examples of this are:
 
 And there are many more examples of this! Game engines strive to be
 general purpose in nature, so balanced algorithms are always favored
-over algorithms that might be the fast in some situations and slow in
+over algorithms that might be the fastest in some situations and slow in
```
Comment (cbscribe) [inline learning/features/3d/3d_performance_and_limitations.rst]: Scans better if you leave it "fast" but remove "the" -> "fast in some ... and slow in others"

**Comment by NathanLovato** [comment]
Thanks for the contribution @jmruzik87 but can't merge it as-is. Do you want to improve your PR so it passes reviews or should I close it?

**Comment by NathanLovato** [comment]
No change to the PR since 2 months, closing it.

**Pull request closed without merging**